manoseimas/settings/base.py

```diff
@@ -82,6 +82,7 @@
 )
 
 COMPRESS_PRECOMPILERS = (
+    ('text/coffeescript', 'coffee --compile --stdio'),
```

Comment:
I might argue that all custom mimetypes should be prefixed with `x-` like (`type/x-coffeescript`) but people on the Internets already do it without the prefix.